lib/node_modules/@stdlib/stats/base/dists/logistic/mgf/benchmark/benchmark.js

```diff
@@ -57,20 +65,26 @@ bench( pkg, function benchmark( b ) {
 
 bench( pkg+':factory', function benchmark( b ) {
 	var mymgf;
+	var len;
 	var mu;
 	var s;
 	var t;
 	var y;
 	var i;
 
-	mu = 10.0;
-	s = 4.0;
+	mu = 0.0;
+	s = 0.5;  // Small s value
 	mymgf = mgf.factory( mu, s );
 
+	len = 100;
```

Comment:
```suggestion
	mymgf = mgf.factory( mu, s );
	len = 100;
```

No need for a new line between initializations.